doc/nrf/protocols/matter/end_product/factory_provisioning.rst

```diff
@@ -140,7 +140,7 @@ The following table lists the parameters of a factory data set:
 | ``rd_uid``             | rotating device ID unique ID         | <16, 32> B           | byte string  | optional    | The unique ID for rotating device ID, which consists of a randomly-generated 128-bit (or longer) octet string.                                                                                                              |
 |                        |                                      |                      |              |             | The rotating device ID is an optional identifier that is used for the :ref:`ug_matter_configuring_ffs` purposes.                                                                                                            |
 |                        |                                      |                      |              |             | This parameter should be protected against reading or writing over-the-air after initial introduction into the device, and stay fixed during the lifetime of the device.                                                    |
-|                        |                                      |                      |              |             | When building an application with the Factory Data support, the `CONFIG_CHIP_FACTORY_DATA_ROTATING_DEVICE_UID_MAX_LEN` must be set with the length of the actual ``rd_uid`` stored in the Factory Data partition.           |
+|                        |                                      |                      |              |             | When building an application with the Factory Data support, the :ref:`CONFIG_CHIP_FACTORY_DATA_ROTATING_DEVICE_UID_MAX_LEN` must be set with the length of the actual ``rd_uid`` stored in the Factory Data partition.      |
```

Comment:
This Kconfig option does not seem to be listed in the config_matter_stack.rst and causes the doc build failure. Needs to be added or just have it between double backticks.